10_deploy_rook.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/bash
+
+git clone https://github.com/rook/rook
+cd rook/cluster/examples/kubernetes/ceph
```

Comment:
This git clone should go in `03_ocp_repo_sync.sh` I think, so it goes in the GOPATH and not in the dev-scripts directory?

Reply:
done